.github/workflows/pr.yaml

```diff
@@ -22,25 +22,22 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 2
-      - uses: actions/setup-go@v4
+      - uses: actions/setup-go@v5
         with:
-          go-version: '1.21'
-      - name: Install dependencies
-        run: go mod download
-      - name: Install GolangCI Lint
-        run: curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.55.2
-
+          go-version-file: 'go.mod'
       - name: golangci-lint
-        run: golangci-lint run --timeout=30m --max-same-issues=0 --out-format=github-actions
+        uses: golangci/golangci-lint-action@v4
+        with:
+          version: v1.55.2
```

Comment:
When using the github action we should have a config file .golangci.yaml to specify max issues and timeouts...

https://github.com/equinor/radix-vulnerability-scanner-api/blob/master/.golangci.yaml

Reply:
added file